### PR TITLE
Page rules

### PR DESCRIPTION
**Pull request by VLeads**
Add Rules page

### CODE REVIEW

File: components/homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.quiz-main-container{
+    padding-top: 3rem
+}
+
+.quiz-main-container > h2{
+    text-align: center
+}
```
Comment (shubhmehra) [inline components/homepage/homepage.css:7]: avoid targeting that element directly use class instead

File: components/rules/rules.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+
+    <link rel="stylesheet" href="./rules.css" />
+    <link rel="stylesheet" href="../../index.css" />
```
Comment (shubhmehra) [inline components/rules/rules.html:18]: you can use relative path over here instead of this absolute imports with ../

File: index.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+.footer > a > i{
+    font-size: 3rem;
+}
+
+.footer-header{
+    font-weight: 600
+}
+
+.footer > a > i:hover{
+    color: #383838
```
Comment (shubhmehra) [inline index.css:72]: use color from root var for consistency

File: components/rules/rules.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.rules-main-container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 4rem;
+}
```
Comment (nayyyhaa) [inline components/rules/rules.css:6]: Just a suggestion, you can create a seperate class for flex & use it wherever required.

File: index.html [hunk 5]
```diff
@@ -1,12 +1,101 @@
...
+                <p class="card-desc">
+                  Want to explore the world of cosmos or to test your knowledge.
+                  Then this is for you !
+                </p>
+              </div>
+            </div>
+          </a>
+          <a href="./components/rules/rules.html">
+            <div class="card-vertical">
+              <img src="./assets/img/tech.jpg" alt="card" />
```
Comment (shubhmehra) [inline index.html:56]: please use valid alt text for  better accessibility

**Review by nayyyhaa — APPROVED** [review]
Just check the points mentioned. Rest nice work! ✅

File: components/rules/rules.html (hunk 2 above)
Comment (VLeads) [reply components/rules/rules.html:18]: I have used absolute path for my component library. How can I use relative path over here ?

File: components/rules/rules.css (hunk 4 above)
Comment (VLeads) [reply components/rules/rules.css:6]: Alright, I'll update it !

File: index.html (hunk 5 above)
Comment (VLeads) [reply index.html:56]: Done

**Pull request merged**